### ISSUE TITLE
Inquiry Regarding the Release of StrongSwan 6.0 with PQC Support

### ISSUE THREAD

**Issue by Dhanasree234** [strongswan/strongswan#2420]
We are currently utilizing the StrongSwan 6.0 beta version for post-quantum cryptography (PQC) support. We would appreciate it if you could kindly provide information regarding the expected release date of the official StrongSwan 6.0 version with PQC support.

Anticipating your positive response 

Thank you for your time and assistance.

**Comment by strongX509** [comment]
We are waiting for IANA to assign the IKEv2 Key Exchange codepoints for the ML-KEM algorithms. Also according to the liboqs forum the final ML-KEM source code isn't available yet since there were some last minute changes relative to the draft standard.

**Comment by strongX509** [comment]
I see that you already contacted the [liboqs project](https://github.com/orgs/open-quantum-safe/s.iscussions/1903) and know that the ML-KEM code isn't ready yet

**Comment by tobiasbrunner** [comment]
Yes, ML-KEM is supported, see the [release notes](https://docs.strongswan.org/docs/latest/news/whatsNew.html).

**Comment by ate-wang** [comment]
> 是的，支持 ML-KEM，请[参阅发行说明](https://docs.strongswan.org/docs/latest/news/whatsNew.html)。

Hello, I would like to ask how to install strongswan into openwrt and compile and test it in openwrt. Is there a corresponding usage process? Can you provide a reference? Looking forward to your reply.

**Comment by Kamal-Kandpal** [comment]
Can someone tell me if the PQC is supported with strongswan, if yes then any example can you share for the PQC configuration.
I want to use strongswan as a server to test with my custom client.
